ui/components/hooks/useStore.js

```diff
@@ -54,5 +54,17 @@ export default function useStore(basePath) {
       update,
     )
 
-  return { state, retryJob, retryAll, selectedStatuses, setSelectedStatuses }
+  const cleanAll = queueName => () =>
+    fetch(`${basePath}/queues/${queueName}/clean`, { method: 'put' }).then(
+      update,
+    )
+
+  return {
+    state,
+    retryJob,
+    retryAll,
+    cleanAll,
+    selectedStatuses,
+    setSelectedStatuses,
+  }
```

Comment:
My code formatter changed this return statement from a single line to multiple lines.  I can change it back if this is an issue.

Reply:
it was probably because a new item was added, so it's fine :D